apps/demo-app/src/app/examples/dialog/error/error-example.component.ts

```diff
@@ -18,6 +18,6 @@ export class ErrorDialogExampleComponent {
           errors: err
         })
       });
-    ;
-  };
+    
+  }
```

Comment:
Is this new typescript feature?

Reply:
lint --fix -> not needed semicolons removed